src/main/kotlin/no/nav/familie/ef/sak/service/steg/VedtaBlankettSteg.kt

```diff
@@ -20,15 +18,21 @@ class VedtaBlankettSteg(private val vedtakService: VedtakService, private val bl
     }
 
     override fun utførSteg(behandling: Behandling, data: VedtakDto) {
-        if (data.resultatType != ResultatType.INNVILGE) {
-            val feilmelding = "Kan ikke sette vedtaksresultat som ${data.resultatType} - ikke implementert"
-            throw Feil(feilmelding, feilmelding, HttpStatus.BAD_REQUEST)
+        when(data) {
+            is Avslå -> {
```

Comment:
Verkar som att samma sak gørs før avslå och innvilget hær? Man kan anvænda `is Avslå, is Innvilget -> {`

Reply:
Ah det var sånn syntaksen var !